terraform/c100-application-deploy.tf

```
@@ -0,0 +1,16 @@
+module "c100-application-deploy" {
+  source     = "./modules/repository-collaborators"
+  repository = "c100-application-deploy"
+  collaborators = [
+    {
+      github_user  = "timja"
+      permission   = "write"
```

```suggestion
      permission   = "Push"
```

---

Bare with on this need to check it is correct

---

would need to be lower case push

---

script is saying to use uppercase atm, so that should be fixed :)
https://github.com/ministryofjustice/github-collaborators/runs/7559608135?check_suite_focus=true

```suggestion
      permission   = "push"
```

---

good point will update that cheers

---

you able to commit the fix here @NickWalt01  ?